### ISSUE TITLE
Improve accessibility with the new Fractal migration

### ISSUE THREAD

**Issue by reemhamz** [mozilla/protocol#815]
## Description
Accessibility with the Fractal site is at an 80% based on Lighthouse's score:
<img width="600" alt="Screen Shot 2022-07-14 at 11 19 46 AM" src="https://user-images.githubusercontent.com/42309026/179017710-526289f2-9fea-47fa-bfa9-33a572f00d3d.png">

Most of the accessibility issues are coming from the menu bar, which should be easy enough to fix.

## Steps to reproduce
- Open Chrome browser
- Open dev tools
- Select `Lighthouse`
- Click `analyze page load`

I'm leaving a screenshot to give a visual of things to fix:

<img width="582" alt="Screen Shot 2022-07-14 at 11 22 13 AM" src="https://user-images.githubusercontent.com/42309026/179018179-76bee4c6-9047-4053-9830-b74ece7ba232.png">
<img width="584" alt="Screen Shot 2022-07-14 at 11 22 35 AM" src="https://user-images.githubusercontent.com/42309026/179018244-14b5202b-e5be-41d1-89e1-88138e3297e4.png">
